homeassistant/components/vizio/__init__.py

```diff
@@ -10,7 +10,7 @@
 import voluptuous as vol
 
 from homeassistant.components.media_player import DEVICE_CLASS_TV
-from homeassistant.config_entries import ENTRY_STATE_LOADED, SOURCE_IMPORT, ConfigEntry
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry, EntryState
```

Comment:
Can we rename it to `ConfigEntryState` since we also use the word "entry" for area, device and entity registry entries.

Reply:
Done.